addons/stock/models/stock_picking.py

```diff
@@ -1064,7 +1064,7 @@ def _sanity_check(self, separate_pickings=True):
 
         if not self._should_show_transfers():
             if pickings_without_moves:
-                raise UserError(_('Please add some items to move.'))
+                raise UserError(_("You can’t validate an empty transfer. Please add some products to move before proceeding."))
```

Comment:
What about making this one “fun”?

Reply:
I don't know I think it's quite fun already :clown_face:

Reply:
This one might stay as it is, because it can definitely appear quite often so we wanted to stay "soft"